src/index.ts

```diff
@@ -76,17 +77,19 @@ const httpProxyMiddleware = async (
       req.url = rewritePath(req.url as string, pathRewrite);
     }
 
-    if (hasRequestBodyMethods.indexOf(req.method as string) >= 0 && typeof req.body === "object") {
-      req.body = JSON.stringify(req.body);
-    }
     proxy
-      .once("proxyReq", ((proxyReq: any, req: any): void => {
-        if (hasRequestBodyMethods.indexOf(req.method as string) >= 0 && typeof req.body === "string") {
-          proxyReq.write(req.body);
-          proxyReq.end();
+      .once("proxyReq", ((proxyReq: http.ClientRequest, req: http.IncomingMessage): void => {
```

Comment:
Hi @CasheeeewNuts 
Thanks for your PR

Serializing `req.body` was used to solve the problem(#3) of sending HTTP Request Body, but it seems that it is no longer needed(#33). 😀

Thanks :)